pkg/envtest/webhook.go

```diff
@@ -69,6 +70,11 @@ type WebhookInstallOptions struct {
 	// LocalServingHostExternalName is the hostname to use to reach the webhook server.
 	LocalServingHostExternalName string
 
+	// CustomTLSConfig is a *tls.Config object that will be used in place of
+	// the one normally created by reading certs in the CertDir.  If both
+	// CertDir and this are specified CertDir will be ignored
+	CustomTLSConfig *tls.Config
```

Comment:
What do you think about just calling this `TLSConfig`? IMO the "Custom" prefix is unnecessary.